src/Components/Facility/AssetCreate.tsx

```diff
@@ -90,7 +90,7 @@ const AssetCreate = (props: AssetProps) => {
   const [asset_class, setAssetClass] = useState<AssetClass>();
   const [not_working_reason, setNotWorkingReason] = useState("");
   const [description, setDescription] = useState("");
-  const [is_working, setIsWorking] = useState("0");
+  const [is_working, setIsWorking] = useState<string | undefined>("");
```

Comment:
```suggestion
  const [is_working, setIsWorking] = useState<string | undefined>(undefined);
```

Initially if this not set to `undefined` the error won't be triggered when we submit the form without modifying is_working